.github/workflows/test.yaml

```diff
@@ -0,0 +1,47 @@
+name: Testing CI pipeline
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build the docker image
+        run: docker-compose build
+
+      - name: up postgresql container runs
+        run: docker-compose up -d
```

Comment:
this will build and run the application already in the docker container. Ideally we only start the "testDb"  container, which I'm sure is possible with a command.

Reply:
It should be "docker compose up -d testDb"

Reply:
Unfortunately, that didn't work..

Reply:
Then please note that as a to-do for later